Keas.Mvc/Services/HistoryService.cs

```diff
@@ -240,18 +240,17 @@ public async Task<History> EquipmentAssigned(Equipment equipment)
             return historyEntry;
         }
 
-        public async Task<History> KeySerialUnassigned(KeySerial keySerial)
+        public async Task<History> KeySerialUnassigned(KeySerialAssignment keySerialAssignment)
         {
             var user = await _securityService.GetUser();
             var historyEntry = new History
             {
-                Description = "Key Unassigned  by " + user.Name,
+                Description = "Key Unassigned by " + user.Name,
                 ActorId = user.Id,
-                AssetType = "Key",
+                AssetType = "KeySerial",
```

Comment:
@jcubbage Can you confirm if the AssetType should be changed here? If it should be, @jpknoll You should probably change it on line 199 above as well.